.circleci/config.yml

```diff
@@ -106,6 +106,14 @@ jobs:
     steps:
       - *attach_workspace
 
+      - run:
+          name: Set crashlytics apk path
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "dev" ]; then
+              echo 'export CRASHLYTICS_APK_PATH=$APK_PATH_STAG' >> $BASH_ENV
+            elif [ "$CIRCLE_BRANCH" = "master" ]; then
```

Comment:
We may like to do a different kind of deploy for master branch, maybe on Google play alpha/beta channel. What do you think? If we do it like that, our own team will only test results if changes go to master branch. Let's discuss with others.

Reply:
Sure.